sys/include/net/gnrc/ipv6/nib/conf.h

```diff
@@ -50,18 +44,15 @@ extern "C" {
 #ifndef CONFIG_GNRC_IPV6_NIB_6LR
 #define CONFIG_GNRC_IPV6_NIB_6LR                      1
 #endif
-#ifndef CONFIG_GNRC_IPV6_NIB_SLAAC
-#define CONFIG_GNRC_IPV6_NIB_SLAAC                    0
```

Comment:
Where is SLAAC disabled for 6lo only nodes?

Reply:
oops, good catch!